tell/data_process_pop_interp.py

```diff
@@ -52,7 +53,7 @@ def merge_mapping_data(map_input_dir, pop_input_dir, start_year, end_year):
      """
     # load FIPS county data for BA number and FIPs code matching for later population sum by BA
     df = pd.DataFrame()
-    for file in os.listdir(map_input_dir):
+    for file in glob(f'{map_input_dir}/*.csv'):
```

Comment:
the previous version was picking up `.log` files in this same dir and trying to process them -- there may be a better way

Reply:
good catch, thanks for resolving that!